Puma.Security.Parser/Options.cs

```diff
@@ -13,6 +13,13 @@
 
 namespace Puma.Security.Parser
 {
+
+    public enum ReportFormat
+    {
+        Text,
```

Comment:
Should we leave this as "MSBuild" in case we add JSON or other formats later?

Reply:
I agree. I think that's a great idea. I'll update Enum.Text to Enum.MSBuild.